queries.py

```diff
@@ -15,3 +15,9 @@ def _filter_active_models(model, column):
     return model.query.filter(
         updated_timestamp >= datetime.utcnow() - AVAILABILITY_TIMEOUT
     )
+
+
+def get_active_nodes_by_service_type(service_type):
```

Comment:
this function is applying filters to query, not executing actual query, so better to rename function from "get" to "filter"